lib/web/app/handler_test.go

```diff
@@ -74,53 +73,70 @@ func hasAuditEventCount(want int) eventCheckFn {
 // TestAuthPOST tests the handler of POST /x-teleport-auth.
 func TestAuthPOST(t *testing.T) {
 	const (
-		stateValue  = "012ac605867e5a7d693cd6f49c7ff0fb"
-		cookieValue = "5588e2be54a2834b4f152c56bafcd789f53b15477129d2ab4044e9a3c1bf0f3b"
+		cookieValue = "5588e2be54a2834b4f152c56bafcd789f53b15477129d2ab4044e9a3c1bf0f3b" // random value we set in the header and expect to get back as a cookie
 	)
 
 	fakeClock := clockwork.NewFakeClockAt(time.Date(2017, 05, 10, 18, 53, 0, 0, time.UTC))
 	clusterName := "test-cluster"
-	publicAddr := "app.example.com"
+	publicAddr := "proxy.teleport.com:443"
```

Comment:
```suggestion
	publicAddr := "proxy.goteleport.com:443"
```

Might as well use a domain we actually own